toolset/benchmark/framework_test.py

```diff
@@ -200,8 +207,11 @@ def tee_output(prefix, line):
 
     for dependency in deps:
       docker_file = os.path.join(self.directory, dependency + ".dockerfile")
-      if not os.path.exists(docker_file):
-        docker_file = os.path.join(docker_dir, dependency + ".dockerfile")
+      if not docker_file or not os.path.exists(docker_file):
+        docker_file = __find(docker_dir, dependency + ".dockerfile")
+      if not docker_file:
+        tee_output(prefix, "Docker build failed; %s could not be found; terminating\n" % (dependency + ".dockerfile"))
+        return 1
```

Comment:
Attempting to address [#13](https://github.com/TechEmpower/FrameworkBenchmarks/issues/3296)